tests/test_database.py

```diff
@@ -32,7 +33,7 @@ async def test_db_should_connect_manually_after_close(db: AioDatabase) -> None:
     await TestModel.aio_create(text='test')
 
     await db.aio_close()
-    with pytest.raises(RuntimeError):
+    with pytest.raises((RuntimeError, OperationalError)):
```

Comment:
Im not sure if its correct behaivour

Reply:
Yeah it's fine for the moment. Psycopg3 raises PoolClosed exception and peewee catches it and reraises it as OperationalError.